.github/workflows/build_wheels.yml

```diff
@@ -45,3 +45,11 @@ jobs:
       with:
         name: energyplus-wheel-${{ matrix.os }}
         path: ./dist
+
+    - name: Deploy on Test PyPi
+      uses: pypa/gh-action-pypi-publish@v1.8.14
+      with:
+        repository-url: https://test.pypi.org/legacy/
+        user: __token__
+        password: ${{ secrets.TESTPYPIPW }}
+        verbose: true
```

Comment:
Added the command to actually deploy the wheel to PyPi, this is such a great step!

Reply:
In a subsequent effort, I'll modify this script to be able to deploy to either PyPi or TestPyPi on demand, as well as for each tag.